ggplot/utils/ggutils.py

```diff
@@ -2,25 +2,87 @@
 """
 from __future__ import (absolute_import, division, print_function,
                         unicode_literals)
-                        
-import matplotlib.pyplot as plt
+
 import json
 import os
 import sys
 
+import matplotlib as mpl
+import matplotlib.pyplot as plt
 import six
 
+from .exceptions import gg_warning, GgplotError, gg_reset
+
+
+if not hasattr(mpl, 'rc_context'):
```

Comment:
This will error as utils has that removes. Also probably not needed anymore with gg_context?